caffe2/ideep/operators/conv_op.cc

```diff
@@ -36,10 +36,7 @@ class IDEEPConvOp : public IDEEPConvPoolOpBase {
     const auto& X = Input(INPUT_X);
     const auto& filter = Input(FILTER);
     auto* Y = Output(OUTPUT);
-    auto grouped = filter.is_grouped() ? 1 : 0;
-    auto Y_dims_conv = CalcOutputDims(
-        X,
-        grouped ? (filter.get_dim(0) * filter.get_dim(1)) : filter.get_dim(0));
+    auto grouped = false;
```

Comment:
Why force `grouped` to `false`?

Reply:
Some background: Group info is now totally hidden inside the ideep. That is, even if a weight is reordered into a tensor with an extra group dimension (`goihw`, `goidwh`, `giohw`, `giodwh`), it will still be regarded as an ordinary tensor without that extra dim. From the fwk side, we don't have to check the group info any more. That's why I set it to false.

Why I keep this line is because I don't want to touch these following checks containing `grouped`. 
```
    CAFFE_ENFORCE(4 == X.ndims());
    CAFFE_ENFORCE(4 == filter.ndims() || (grouped && (group_ > 1)));
    CAFFE_ENFORCE_EQ(filter.get_dim(2 + grouped), kernel_h());
    CAFFE_ENFORCE_EQ(filter.get_dim(3 + grouped), kernel_w());
    CAFFE_ENFORCE(
        X.get_dim(1) == filter.get_dim(1 + grouped) * group_,
        "Convolution op: input channels does not match: # of input channels ",
        X.get_dim(1),
        " is not equal to kernel channels * group:",
        filter.get_dim(1 + grouped),
        "*",
        group_);
```

That being said, if you consider it's better to remove them, I'd like to change it.